.github/workflows/ci.yml

```diff
@@ -155,28 +155,16 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v3
-    - name: Configure local host
-      run: |
-        python3 -m pip install h2
-        cd ./tests/py_localhost/
-        python3 server.py &
-        python3 non_tls_server.py &
     - name: Build and test
       run: |
         python3 -c "from urllib.request import urlretrieve; urlretrieve('${{ env.BUILDER_HOST }}/${{ env.BUILDER_SOURCE }}/${{ env.BUILDER_VERSION }}/builder.pyz?run=${{ env.RUN }}', 'builder.pyz')"
-        python3 builder.pyz build -p aws-c-http --cmake-extra=-DENABLE_LOCALHOST_INTEGRATION_TESTS=ON
```

Comment:
why doesn't `localhost-test-linux` require `-DENABLE_LOCALHOST_INTEGRATION_TESTS=ON` anymore?

that `ENABLE_LOCALHOST_INTEGRATION_TESTS` is still set in the `localhost-test-mac` and `localhost-test-win` runs

Reply:
was trying to find out why the test coverage report different lines covered.